client/src/main/scala/org/scalaexercises/evaluator/EvaluatorClient.scala

```diff
@@ -7,38 +7,21 @@
 
 package org.scalaexercises.evaluator
 
-import cats.data.EitherT
-import cats.~>
-import cats.implicits._
-import org.scalaexercises.evaluator.EvaluatorResponses.{
-  EvalIO,
-  EvaluationException,
-  EvaluationResponse,
-  EvaluationResult
-}
-import org.scalaexercises.evaluator.free.algebra.EvaluatorOp
-
-import scala.concurrent.Future
-import scala.concurrent.ExecutionContext.Implicits.global
-
-class EvaluatorClient(url: String, authKey: String) {
+import cats.effect.{ConcurrentEffect, Resource}
+import org.http4s.client.Client
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.scalaexercises.evaluator.service.{HttpClientHandler, HttpClientService}
 
-  lazy val api: EvaluatorAPI[EvaluatorOp] = new EvaluatorAPI(url, authKey)
+import scala.concurrent.ExecutionContext
 
-}
+case class EvaluatorClient(url: String, authKey: String)
```

Comment:
Probably this is your problem, you might not need this case class
```suggestion

```

Reply:
Tried without the case class and nothing